### PR TITLE
DRAFT - Add Mandiant Advantage Threat Intelligence

### PR DESCRIPTION
**Pull request by chrishultin**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Expanding the integration with Mandiant Advantage

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MandiantAdvantageThreatIntelligence/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Mandiant Advantage Threat Intelligence",
+    "description": "Integrate your Mandiant Advantage Threat Intelligence data with Cortex XSOAR",
+    "support": "developer",
```
Comment (kgal-pan) [inline Packs/MandiantAdvantageThreatIntelligence/pack_metadata.json:4]:
```suggestion
    "support": "partner",
```

**Comment by kgal-pan** [comment]
@chrishultin - Please remove the file `MATI.zip/uploadable_packs/MandiantAdvantageThreatIntelligence.zip`.

**Comment by kgal-pan** [comment]
@chrishultin - Any updates on this item or can we close? It's been stale for 1.5 months now...

**Comment by chrishultin** [comment]
You can go ahead and close this for now.  I'm still waiting on corporate
legal approval to sign the CLA, unfortunately.

**Pull request closed without merging**